### PR TITLE
Test systemd cgroup manager

### PR DESCRIPTION
**Pull request by runcom**
Tests for https://github.com/kubernetes-incubator/cri-o/pull/524

@mrunalp PTAL

### CODE REVIEW

File: test/pod.bats [hunk 1]
```diff
@@ -287,3 +287,43 @@ function teardown() {
...
+	fi
+
+	wrong_cgroup_parent_config=$(cat "$TESTDATA"/sandbox_config.json | python -c 'import json,sys;obj=json.load(sys.stdin);obj["linux"]["cgroup_parent"] = "podsandbox1.slice:container:infra"; json.dump(obj, sys.stdout)')
+	echo "$wrong_cgroup_parent_config" > "$TESTDIR"/sandbox_wrong_cgroup_parent.json
+
+	start_crio
+	run crioctl pod run --config "$TESTDIR"/sandbox_wrong_cgroup_parent.json
+	echo "$output"
+	[ "$status" -eq 1 ]
+	[[ "$output" =~ "expected cgroupsPath to be of format \"slice:prefix:name\" for systemd cgroups" ]]
```
Comment (mrunalp) [inline test/pod.bats]: We can't check this as it is runtime specific.

Comment (mrunalp) [inline test/pod.bats]: Actually might be okay since we are gating it on system cgroup manager.

**Comment by runcom** [comment]
retest

File: test/pod.bats (hunk 1 above)
Comment (mrunalp) [inline test/pod.bats]: @sameo could you take a look?

**Comment by runcom** [comment]
retest

File: test/pod.bats (hunk 1 above)
Comment (runcom) [inline test/pod.bats]: I think it makes sense to drop this condition for $output and just check that status is != 0 as other runc-like runtimes may have different output..

Comment (sameo) [inline test/pod.bats]: @runcom @mrunalp yes, this is a libcontainer specific string so we probably want to only check the exit status of the command.

Comment (runcom) [inline test/pod.bats]: Awesome I'll do that then

Comment (runcom) [inline test/pod.bats]: fixed

**Comment by runcom** [comment]
This should be good to go now.

**Comment by mrunalp** [comment]
We did not get the e2e results for Fedora.

**Comment by mrunalp** [comment]
LGTM

**Comment by runcom** [comment]
@sameo PTAL

**Comment by sameo** [comment]
LGTM

**Pull request merged**